src/test/scala/org/broadinstitute/dsde/firecloud/dataaccess/MockGroupSupport.scala

```diff
@@ -0,0 +1,15 @@
+package org.broadinstitute.dsde.firecloud.dataaccess
+
+import org.broadinstitute.dsde.workbench.model.{WorkbenchEmail, WorkbenchGroupName}
+
+/**
+  * Created by mbemis on 4/25/18.
+  */
+trait MockGroupSupport {
```

Comment:
this only exists because the endpoints are sticking around in rawls for now but there's still one endpoint related to groups that we need in sam. this lets the tests share a common group map